### PR TITLE
Feature/sscs 2226 form download

### PR DESCRIPTION
**Pull request by iguannago**
Add the new appointee form download step: controller, template and content file
Add unit and e2e coverage
A few refactoring stuff because I was not clear whether this was an ExitPoint or a Question Type class. 
I use i18next to interpolate the content with the nunjuck variable in it and render it in the e2e for testing purpose, however, I finally decided to leave the literal text because of simplicity.

### CODE REVIEW

File: steps/identity/appointee-form-download/AppointeeFormDownload.js [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
-class AppointeeFormDownload extends ExitPoint {
+class AppointeeFormDownload extends Question {
 
     static get path() {
 
         return paths.identity.downloadAppointeeForm;
     }
 
-    get form() {}
+    get benefitType() {
```
Comment (paulgain) [inline steps/identity/appointee-form-download/AppointeeFormDownload.js]: Space

File: steps/identity/appointee-form-download/AppointeeFormDownload.js [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+
+    get form() {
 
+        return form(
+
+            textField.ref(this.journey.steps.BenefitType, 'benefitType')
+        );
+    }
 
     next() {}
```
Comment (paulgain) [inline steps/identity/appointee-form-download/AppointeeFormDownload.js]: There's no need to defined next() in this instance, you can safely remove it.

File: steps/identity/appointee-form-download/AppointeeFormDownload.js (hunk 1 above)
Comment (iguannago) [reply steps/identity/appointee-form-download/AppointeeFormDownload.js]: done

File: steps/identity/appointee-form-download/AppointeeFormDownload.js (hunk 2 above)
Comment (iguannago) [reply steps/identity/appointee-form-download/AppointeeFormDownload.js]: done

File: test/unit/steps/identity/appointee-form-download/AppointeeFormDownload.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const AppointeeFormDownload = require('steps/identity/appointee-form-download/AppointeeFormDownload');
+const {expect} = require('test/util/chai');
+const paths = require('paths');
+
+describe('AppointeeFormDownload.js', () => {
+
+    let appointeeFormDownloadSut;
```
Comment (paulgain) [inline test/unit/steps/identity/appointee-form-download/AppointeeFormDownload.test.js]: For consistency please remove `Sut` from the class name, we haven't adopted that approach and to be honest it doesn't add any value.

Comment (iguannago) [reply test/unit/steps/identity/appointee-form-download/AppointeeFormDownload.test.js]: done

**Pull request merged**